designsafe/static/scripts/data-depot/components/published/published.component.html

```diff
@@ -166,7 +166,7 @@ <h2>{{$ctrl.project.value.projectId}}: {{ $ctrl.project.value.title }}</h2>
             <span ng-repeat-end class="tag-title">{{entity.value.title}}</span>
             </div>
             </td>
-            <td ng-if="item.type == 'dir'">--</td>
+            <td ng-if="item.type == 'dir'">&nbsp;</td>
```

Comment:
Is this an actual requirement? If not, usually they don't like empty spaces and it's better to show `--`